src/lib/php/common-sysconfig.php

```diff
@@ -218,6 +218,13 @@ function Populate_sysconfig()
   $valueArray[$variable] = array("'$variable'", "'localhost:3000'", "'$prompt'",
     strval(CONFIG_TYPE_TEXT), "'PAT'", "3", "'$desc'", "null", "null");
 
+  /*  Expose Headers */
+  $variable = "Headers";
+  $prompt = _('Allowed headers for REST API');
+  $desc = _('"*" for anywhere');
+  $valueArray[$variable] = array("'$variable'", "'x-total-pages'", "'$prompt'",
```

Comment:
I don't think it needs to be a configuration. In order for API to function properly, the admin any way needs to allow all the headers. It will be better to have it in code.

Reply:
Okay so will pass the wildcard operator